datastore/src/main/java/com/openlattice/datastore/directory/controllers/PrincipalDirectoryController.java

```diff
@@ -146,6 +148,17 @@ public User getUser( @PathVariable( USER_ID ) String userId ) {
         return userDirectoryService.getUser( userId );
     }
 
+    @Timed
+    @RequestMapping(
+            path = USERS,
+            method = RequestMethod.POST,
+            consumes = MediaType.APPLICATION_JSON_VALUE,
+            produces = MediaType.APPLICATION_JSON_VALUE
+    )
+    public Map<String, User> getUsers( @Body Set<String> userIds ) {
```

Comment:
This annotation should be `@RequestBody`

Reply:
Is there any documentation on why we use retrofit.* on APIs and springframework.* for implementations?

Reply:
I didn't even know it was different packages.